.circleci/config.yml

```diff
@@ -185,7 +185,30 @@ jobs:
     steps:
       - checkout
       #- run: git clone https://github.com/openmf/ph-ee-integration-test
-      - run: ./gradlew test -Dcucumber.filter.tags="@amsIntegration"
+      - run:
+          name: Ngrok setup
+          command: curl -s https://ngrok-agent.s3.amazonaws.com/ngrok.asc | sudo tee /etc/apt/trusted.gpg.d/ngrok.asc >/dev/null && echo "deb https://ngrok-agent.s3.amazonaws.com buster main" | sudo tee /etc/apt/sources.list.d/ngrok.list && sudo apt update && sudo apt install ngrok
+      - run:
+          name: Test execution
+          command: |
+            ngrok http 53013 > /dev/null &
+            echo -n "Extracting ngrok public url ."
+            NGROK_PUBLIC_URL=""
+            while [ -z "$NGROK_PUBLIC_URL" ]; do
+              # Run 'curl' against ngrok API and extract public (using 'sed' command)
+              export NGROK_PUBLIC_URL=$(curl --silent --max-time 10 --connect-timeout 5 \
+                                        --show-error http://127.0.0.1:$LOCAL_PORT/api/tunnels | \
+                                        sed -nE 's/.*public_url":"https:..([^"]*).*/\1/p')
+              sleep 1
+              echo -n "."
+            done
+
+            export CALLBACK_URL="https://$NGROK_PUBLIC_URL"
+            echo -n "Public url ."
+            echo $CALLBACK_URL
+            ./gradlew test -Dcucumber.filter.tags="@amsIntegration"
+            echo -n "Test execution is completed, kill ngrok"
+            pkill ngrok
       - store_test_results:
           path: build/test-results/test/TEST-org.mifos.integrationtest.TestRunner.xml
```

Comment:
please check the store test results part. the results are not populated in your ams test after this PR

Reply:
issue reported.  Not a side effect of this PR